okhttp/src/main/java/okhttp3/internal/ws/RealWebSocket.java

```diff
@@ -163,6 +177,11 @@ public RealWebSocket(Request request, WebSocketListener listener, Random random,
     };
   }
 
+  public RealWebSocket(Request request, WebSocketListener listener, Random random,
```

Comment:
I’m looking forward to using default parameters

Reply:
Yep will simplify a bit.